downstreamadapter/eventcollector/event_collector.go

```diff
@@ -421,6 +445,7 @@ func (c *EventCollector) updateMetrics(ctx context.Context) error {
 		case <-ctx.Done():
 			return context.Cause(ctx)
 		case <-ticker.C:
+			// FIXME: record ds?
```

Comment:
how about here now?

Reply:
the redo metrics will be updated in the future.